dipy/tracking/utils.py

```diff
@@ -921,13 +921,16 @@ def unique_rows(in_array, dtype='f4'):
 
 
 @_with_initialize
-def move_streamlines(streamlines, output_space, input_space=None):
+def move_streamlines(streamlines, output_space, input_space=None,
+                     return_seeds=False):
     """Applies a linear transformation, given by affine, to streamlines.
 
     Parameters
     ----------
     streamlines : sequence
         A set of streamlines to be transformed.
+        If return_seeds is True, `streamlines` will be a tuple
+        of (streamlines, seeds)
     output_space : array (4, 4)
```

Comment:
We need an additional entry here for `return_seeds`. In particular, it needs to be spelled out that it's optional and that the default is False.

Reply:
This is indeed weird, not sure why I left it that way. Will change asap !

Reply:
Changed it so seeds have their own field, should be less awkward now !